.circleci/config.yml

```diff
@@ -174,24 +174,12 @@ workflows:
       - docs:
           requires:
             - build
-      - docs-master:
```

Comment:
Should you also delete https://github.com/adobe-private/react-spectrum-v3/blob/cf5410adaedecec6a6dacfe1b7b7f556973bbbb2/.circleci/config.yml#L104-L121 ?